tests/puppeteer/campaigns/sanity/products/02_CRUD_simple_product_in_BO

```diff
@@ -0,0 +1,83 @@
+// Using chai
```

Comment:
Aren't you missing the extension for this file ?

Reply:
done :+1: